.github/workflows/pre-release.yaml

```diff
@@ -0,0 +1,66 @@
+# Workflow to fetch the latest commit hash on the main branch and upload artifacts to CF storage.
```

Comment:
`# Build Stellar contracts on every commit to main and upload the wasm code to Cloudflare R2